CMakeLists.txt

```diff
@@ -0,0 +1,51 @@
+cmake_minimum_required(VERSION 3.13.0)
```

Comment:
What determines this? Is this actually not compatible with older versions of CMake?

Reply:
Decreased the minimum to 3.1.0 (AFAIK the first CMake version that supports "modern" (target-based) cmake).